h3i/src/main.rs

```diff
@@ -202,6 +202,26 @@ fn config_from_clap() -> std::result::Result<Config, String> {
                 .requires("qlog-input")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("disable-dgram")
```

Comment:
This option is a bit confusing. The datagram extension is optional and endpoints opt in to **receiving** datagrams from their peer, not sending them to their peer. 

It would be clearer for the default for h3i to be datagram reception disabled, and allow users to opt in to receiving them i.e., change this to `enable-dgram` and `Enable datagram reception`